packages/verify-provenance/src/index.ts

```diff
@@ -1,6 +1,5 @@
 // TODO
-export const verifyProvenance = (
-  bundle: object,
-): boolean => {
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
This seems unrelated.

Reply:
addresses a linter error, the CI breaks without it

Reply:
`CI` was red even 8 months ago it seems.  Let's ONLY change the package-lock here and let the codeowners fix linting next time they want to use this repo.